docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sample for ModelMesh Serving using the KServe Python SDK\n",
+    "\n",
+    "## This notebook shows how to use the KServe SDK to create, update, and delete an InferenceService\n",
+    "\n",
+    "- This sample assumes ModelMesh Serving was deployed using the [quickstart guide](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kubernetes import client\n",
+    "from kserve import constants\n",
+    "from kserve import V1beta1InferenceService\n",
+    "from kserve import V1beta1InferenceServiceSpec\n",
+    "from kserve import V1beta1PredictorSpec\n",
+    "from kserve import V1beta1SKLearnSpec\n",
+    "from kserve import V1beta1TFServingSpec\n",
+    "from kserve import V1beta1StorageSpec\n",
+    "from kserve import KServeClient"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The variables being set are `namespace`, `name`, `protocol_version`\n",
+    "- The `namespace` definition is where the InferenceService will be deployed to\n",
```

Comment:
InferenceService should also be in-line code like `InferenceService` otherwise spell it in two words "Inference Service" -- @rafvasq what do you think?

Reply:
Yup I agree. I think it helps to make it more obvious that we're referring to the custom resource and not anything else. @lgdeloss, this would apply all around including headers like [this one](https://github.com/kserve/modelmesh-serving/pull/395/files#diff-9db730d6dd336df2fba42635af7136fa0b5190874583c43bae658aa07429332eR57)